src/components/emailerFilterControls/index.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useCallback, useState, useEffect } from 'react';
+import moment from 'moment';
+import { EmailerFilters } from '../../containers/email/types';
+import {
+  Collapse,
+  Slider,
+  DatePicker,
+  AutoComplete,
+  Button,
+  message,
+} from 'antd';
+import { SliderMarks } from 'antd/lib/slider';
+import styled from 'styled-components';
+import { Neighborhoods } from '../../assets/content';
+import apiClient from '../../api/apiClient';
+import { CloseOutlined } from '@ant-design/icons';
+
+const StyledCollapse = styled(Collapse)`
+  max-width: 400px;
+`;
+
+const StyledRangePicker = styled(DatePicker.RangePicker)`
+  margin-right: 15px;
+`;
+
+const autoCompleteStyles = { width: 200, marginRight: 15 };
+
+interface EmailerFilterControlsProps {
+  filters: EmailerFilters;
+  setFilters: React.Dispatch<React.SetStateAction<EmailerFilters>>;
+}
+
+const MAX_COUNT = 10;
+
+function activityCountRange(filters: EmailerFilters): [number, number] {
+  return [filters.activityCountMin, filters.activityCountMax || MAX_COUNT + 1];
+}
+
+function formatDates(
+  start?: string,
+  end?: string,
+): [moment.Moment | null, moment.Moment | null] | undefined {
+  if (start && end) return [moment(start), moment(end)];
+}
+
+function disabledDate(current: moment.Moment): boolean {
+  // Can not select future days
+  return current > moment().endOf('day');
+}
+
+const activityCountLabels: SliderMarks = {
+  0: '0',
+  [MAX_COUNT + 1]: `${MAX_COUNT}+`,
+};
+
+const neighborhoodOptions = Object.values(Neighborhoods)
```

Comment:
🔧 Noticed that the neighborhood names weren't being sorted and I think it's because we're trying to sort an array of objects so we probably need to pass in a custom function for `sort()`. The easier alternative tho I think is just sorting before mapping (i.e. `const neighborhoods = Object.values(Neighborhoods).sort().map((value) => ({ value }));`)